google/cloud/bigtable/data/_async/mutations_batcher.py

```diff
@@ -328,17 +331,26 @@ async def _flush_internal(self, new_entries: list[RowMutationEntry]):
         """
         # flush new entries
         in_process_requests: list[asyncio.Future[list[FailedMutationEntryError]]] = []
+        metric = self._table._metrics.create_operation(OperationType.BULK_MUTATE_ROWS)
```

Comment:
We don't have this OperationType in java 😓  let's just use MUTATE_ROWS for this.

Reply:
BULK_MUTATE_ROWS is just the name we've been using for mutate_rows, since it's easier to parse. The string value is still "MutateRows"

Reply:
Gotcha!